db/migrate/20160124145702_create_comments.rb

```diff
@@ -0,0 +1,12 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table :comments do |t|
+      t.integer :user_id
+      t.string :content
+      t.integer :votes
+      t.references :comment_on, polymorphic: true, index: true
```

Comment:
Not much of a comment, but a convention thingy! Why using comment_on? Thought it would be clearer for anyone taking on the project to see commentable(has_many comments as: :commentable and belongs_to  :commentable)